src/main/java/mate/academy/car/sharing/controller/UserController.java

```diff
@@ -0,0 +1,50 @@
+package mate.academy.car.sharing.controller;
+
+import lombok.RequiredArgsConstructor;
+import mate.academy.car.sharing.dto.request.UserRequestDto;
+import mate.academy.car.sharing.dto.response.UserResponseDto;
+import mate.academy.car.sharing.entity.User;
+import mate.academy.car.sharing.mapper.UserMapper;
+import mate.academy.car.sharing.service.UserService;
+import org.springframework.security.core.Authentication;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/users")
+public class UserController {
+    private final UserService userService;
+    private final UserMapper mapper;
+
+    @GetMapping("/me")
+    public UserResponseDto get(Authentication auth) {
+        String email = auth.getName();
+        User user = userService.findByEmail(email).orElseThrow(
+                () -> new RuntimeException("User with email " + email + " not found"));
```

Comment:
Better use custom exception (NoSuchElement or create your own Exception NoSuchUser for example)